source/pdal/pdalc_dimtype.cpp

```diff
@@ -26,11 +26,13 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  *****************************************************************************/
-
+#define _CRT_SECURE_NO_WARNINGS
```

Comment:
This is usually done with compile-time definitions instead of sticking it here, but i'm not much of a windows dev